staging/src/k8s.io/apiserver/pkg/audit/policy/reader_test.go

```diff
@@ -52,7 +52,7 @@ rules:
 `
 
 const policyDefV1beta1 = `
-apiVersion: audit.k8s.io/v1beta1
+apiVersion: audit.k8s.io/v1
```

Comment:
Don't do this like that, we need to ensure we're reading previous versions. You need to add `policyDefV1` and ensure that the new and old ones are properly readable.

Reply:
Done.